d2bs/kolbot/tools/Party.js

```diff
@@ -130,17 +130,19 @@ function main() {
 							break;
 						}
 					case 2: // Accept invites
-						if (Config.Leader && player.name !== Config.Leader) {
-							break;
-						}
+						if (myPartyId === 65535) {
+							if (Config.Leader && player.name !== Config.Leader) {
+								break;
+							}
 
-						if (player.partyid !== 65535 && player.partyid !== myPartyId) {
-							otherParty = player.partyid;
-						}
+							if (player.partyid !== 65535 && player.partyid !== myPartyId) {
```

Comment:
the first part of that check isn't possible since its inside the if statement checking the opposite

Reply:
the if on line 133 checks myPartyId which is set on line 99. the if on line 138 is checking someone else's partyid.

Reply:
to be honest, I have no idea what the `otherParty` check is even for. I think we can remove it completely because on line 142, if partyflag is 2 that means they already invited you, which implies they are allowed to invite you.

Reply:
hmm okay then it seems it is redundant 

myPartyId is always 65535

so != myPartyId is the same as the first part